app/build.gradle

```diff
@@ -645,7 +646,7 @@ dependencies {
     geckoImplementation deps.gecko_view."${branch}_arm64"
     configurations.all {
          resolutionStrategy.capabilitiesResolution.withCapability('org.mozilla.geckoview:geckoview') {
-            def abi = getName().toLowerCase().contains('x86_64') ? 'x86_64' : 'arm64'
+            def abi = getName().toLowerCase().contains('x64') ? 'x86_64' : 'arm64'
```

Comment:
s/x86_64/x64 ?

Reply:
nop, because this is used to get the candidate name for the geckoview build, and it uses x86_64